scaffolding-chef-infra/tests/user-linux-api/tests/test.bats

```diff
@@ -15,6 +15,11 @@ SCAFFOLD_PKG_CHEF_CLIENT_VERSION="$(echo "${scaffold_chef_client}" | cut -d/ -f3
   [ "${result}" = "${scaffold_cacerts}" ]
 }
 
+@test "API: rubygems_url matches default.toml" {
+  result="$(grep '^cfg_rubygems_url = .*' /hab/svc/${TEST_PKG_NAME}/config/client-config.rb | cut -d ' ' -f3)"
```

Comment:
```suggestion
  result="$(grep '^rubygems_url = .*' /hab/svc/${TEST_PKG_NAME}/config/client-config.rb | cut -d ' ' -f3)"
```

I believe this will fix the test

Reply:
If I remember correctly we had tried that. the issue is that when we run that locally we get the expected URL result but it seems to be that when it is run through bats it is not getting what we expect. I think this will pull in the wrong line though for the grep command and will also fail as the result I think will be `#{cfg_rubygems_url}` and not `https://www.mock-rubygems.org` we are looking for.